packages/app/src/cli/models/app/loader.ts

```diff
@@ -44,7 +46,7 @@ async function loadConfigurationFile(
 
   try {
     const configurationContent = await readFile(filepath)
-    return decode(configurationContent)
+    return extname(filepath) === '.json' ? JSON.parse(configurationContent) : decode(configurationContent)
```

Comment:
naive implementation, we'd put middleware here to decide if/when/how to encode/decode. could support yaml too pretty easily